packages/server/src/migration.ts

```diff
@@ -0,0 +1,10 @@
+import * as migrateMongo from "migrate-mongo";
+import * as config from "../migrate-mongo-config.js";
+
+export async function migrate() {
```

Comment:
I don't like that this function uses a different data base connection than the main program. But there are some type errors, where it seems that migrate-mongo adds some stuff to the client and db object.